src/scripts.js

```diff
@@ -163,13 +163,13 @@ function addWeeklyStepInfo(id, activityInfo, dateString, userStorage, laterDateS
   $('#bestUserSteps').prepend(makeStepsHTML(user, activityInfo, userStorage, activityInfo.userDataForWeek(winnerId, dateString, userStorage, "numSteps")))
 }
 
-function addUserTodayStepDetails(id, activityInfo, dateString, userStorage, laterDateString, user, winnerId) {
+function addUserTodayStepDetails(id, activityInfo, dateString, userStorage) {
   $('#userStairsToday').prepend(`<p>Stair Count:</p><p>You</><p><span class="number">${activityInfo.userDataForToday(id, dateString, userStorage, 'flightsOfStairs')}</span></p>`)
   $('#userStepsToday').prepend(`<p>Step Count:</p><p>You</p><p><span class="number">${activityInfo.userDataForToday(id, dateString, userStorage, 'numSteps')}</span></p>`)
   $('#userMinutesToday').prepend(`<p>Active Minutes:</p><p>You</p><p><span class="number">${activityInfo.userDataForToday(id, dateString, userStorage, 'minutesActive')}</span></p>`)
 }
 
-function addUserAvgStepDetails(id, activityInfo, dateString, userStorage, laterDateString, user, winnerId) {
```

Comment:
Looks so much better! We need to look at all these functions later to see if we can refactor them more. Many of them look pretty repetitive.